### PR TITLE
Minor tweak to gitpod

### PR DESCRIPTION
**Pull request by timja**
recommended in https://github.com/jenkinsci/design-library-plugin/pull/63#discussion_r875961782

hopefully it fixes the 'open preview' being broken in intellij

<a href="https://gitpod.io/#https://github.com/jenkinsci/design-library-plugin/pull/66"></a>

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 tasks:
-  - init: |
-      mvn install -P quick-build
+  - init: mvn install -P quick-build
```
Comment (timja) [inline .gitpod.yml:2]: @akosyakov not sure if you will know but for some reason I get an internal server error on preview:

> HISTFILE=/workspace/.gitpod/cmd-1 history -r; {
> [ -r /workspace/.prebuild-log-1 ] && cat /workspace/.prebuild-log-1; [ -r /workspace/.gitpod/prebuild-log-1 ] && cat /workspace/.gitpod/prebuild-log-1; true
> } && {
> gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
> }
Awaiting port 8080... ok
https://8080-jenkinsci-designlibrary-cvykmhpwggp.ws-eu45.gitpod.io
2022/05/18 17:51:27 500 Internal Server Error


image

Copying the command into my terminal and running it is just fine:

>  gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/

Comment (akosyakov) [reply .gitpod.yml:2]: hm, it is probably bug on our side we need to wait till JB backend is running.

Could you try to add  `gp await-port 63342`  as a workaround?

File: .gitpod.yml [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+    command: |
       curl https://gist.githubusercontent.com/timja/fe73f0a0f98afe4d9356e7ce6752d95b/raw/a8d772267bed81fb0c11af2c69eaa71ebc7daec0/init.sh | bash
-    command: mvn hpi:run
+      mvn hpi:run
+    name: Run
   - command: gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
```
Comment (timja) [inline .gitpod.yml:7]:
```suggestion
  - command: gp await-port 63342 && gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
```

File: .gitpod.yml (hunk 1 above)
Comment (timja) [reply .gitpod.yml:2]: that fixed it thanks!

File: .gitpod.yml [hunk 3]
```diff
@@ -1,9 +1,11 @@
 tasks:
-  - init: |
-      mvn install -P quick-build
+  - init: mvn install -P quick-build
+    command: |
       curl https://gist.githubusercontent.com/timja/fe73f0a0f98afe4d9356e7ce6752d95b/raw/a8d772267bed81fb0c11af2c69eaa71ebc7daec0/init.sh | bash
-    command: mvn hpi:run
-  - command: gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
+      mvn hpi:run
```
Comment (timja) [inline .gitpod.yml:5]:
```suggestion
      mvn hpi:run -Dhost=0.0.0.0
```

File: .gitpod.yml (hunk 1 above)
Comment (timja) [reply .gitpod.yml:2]: It worked the first time but failed the second time:

```
gitpod /workspace/design-library-plugin (timja-patch-1) $  HISTFILE=/workspace/.gitpod/cmd-1 history -r; {
> [ -r /workspace/.prebuild-log-1 ] && cat /workspace/.prebuild-log-1; [ -r /workspace/.gitpod/prebuild-log-1 ] && cat /workspace/.gitpod/prebuild-log-1; true
> } && {
> gp await-port 63342 && gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
> }
Awaiting port 63342... ok
Awaiting port 8080... ok
https://8080-jenkinsci-designlibrary-pigv2mnet01.ws-eu45.gitpod.io
2022/05/19 19:43:01 500 Internal Server Error

```

Comment (akosyakov) [reply .gitpod.yml:2]: We are looking in permanent fix, most likely will be out beginning next week.

**Pull request merged**